spec/api/v1/admin/diagnoses_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'rails_helper'
+
+describe V1::Admin::Diagnoses do
+  include AuthHelper
+
+  let!(:current_user) { create(:user) }
+  let!(:diagnoses) { create_list(:diagnosis, 2) }
+
+  describe 'POST' do
```

Comment:
`POST` is not accurate enough in that context, you can have a couple of handlers that manage post queries, so this description doesn't suit here